providers/edge3/docs/commits.rst

```diff
@@ -33,131 +33,16 @@ This is detailed commit list of changes for versions provider package: ``edge3``
 For high-level changelog, see :doc:`package information including changelog <index>`.
 
 
+
 1.0.0
 .....
 
-Latest change: 2025-04-16
+Latest change: 2025-04-19
 
-==================================================================================================  ===========  ============================================================================================================================================================
+==================================================================================================  ===========  ==========================================================================
 Commit                                                                                              Committed    Subject
-==================================================================================================  ===========  ============================================================================================================================================================
+==================================================================================================  ===========  ==========================================================================
+`223a741869 <https://github.com/apache/airflow/commit/223a741869505ad31c38310f307bf2f0f0f193fb>`__  2025-04-19   ``capitalize the term airflow (#49450)``
+`2ae12c815b <https://github.com/apache/airflow/commit/2ae12c815bc704eff6890df56f7387da513d14f2>`__  2025-04-16   ``Update documentation for edge3 and git provider (#49365)``
 `99666f9789 <https://github.com/apache/airflow/commit/99666f97893dff28297555778620a9c114779667>`__  2025-04-16   ``Rename 'edge' provider fom 'edgeexecutor' to 'edge3' provider (#49358)``
-`ed93c3f697 <https://github.com/apache/airflow/commit/ed93c3f6970eb207c727b2f4a1da295fa4c74cc3>`__  2025-04-05   ``fix: use pydantic model for edge worker registration response (#48821)``
```

Comment:
Is this a "bug" that all commit history is removed? (because of the name change?)

Reply:
probably due to the provider rename. we can look into this later